src/nxt_main.c

```diff
@@ -5,12 +5,17 @@
  */
 
 #include <nxt_main.h>
+#include <nxt_otel.h>
```

Comment:
Ignore that, see above....

Reply:
Yep, the otel header shouldn't be in `nxt_main.c`.
Btw, I'm a bit confused when I look into the `nxt_otel.c`, since I thought it only has anything to do with otel infrastructure, but it also depend on http request.
Anyway it's better to only include it in one place.